cfgov/unprocessed/root/financial-intuition.xml

```diff
@@ -16,6 +16,51 @@
         />
         <itunes:category text="Education"></itunes:category>
         <itunes:explicit>false</itunes:explicit>
+        <item>
+            <title>Managing Your Student Loans, Part 3: Postgraduate Degree Repayment Options</title>
+            <link>https://www.consumerfinance.gov/practitioner-resources/students/financial-intuition/managing-your-student-loans-part-3/</link>
+            <itunes:episodeType>full</itunes:episodeType>
+            <itunes:episode>6</itunes:episode>
+            <description>For the third and final episode of this series, we spoke with Dr. Bridget Peters, a family medicine resident, about tips and best practices for managing money and postgraduate student loan debt as a young professional.</description>
+            <enclosure
+                length="9393384"
+                type="audio/mpeg"
+                url="https://files.consumerfinance.gov/f/media/cfpb_financial-intuition-podcast_managing-your-student-loans_episode-3.mp3"
+            />
+            <pubDate>Tue, 15 Dec 2020 16:00:00 EDT</pubDate>
```

Comment:
For the last batch of three episodes, I had to add a second to the publication date to make sure podcast apps showed them in the right order. Also, EST instead of EDT.

```suggestion
            <pubDate>Tue, 15 Dec 2020 16:00:02 EST</pubDate>
```